doc/api/fs.md

```diff
@@ -1872,10 +1879,16 @@ object with an `encoding` property specifying the character encoding to use for
 the path passed to the callback. If the `encoding` is set to `'buffer'`,
 the path returned will be passed as a `Buffer` object.
 
+*Note*: If `path` resolves to a Socket or a Pipe, the function will return the
```

Comment:
I am not sure what it should be called, before my patch `echo "" | node -e "require('fs').realpath('/dev/stdin', console.log)"` was raising error but this my patch now it has a similar result with python, `/proc/12938/fd/pipe:[224961]`